code/modules/admin/secrets.dm

```diff
@@ -145,6 +146,13 @@
 				for(var/datum/disease/D in SSdisease.processing)
 					D.cure(D)
 
+		if("reset_name")
+			if(!check_rights(R_ADMIN))
+				return
+			world.name = "Space Station 13"
```

Comment:
This should either use the initial name of the station or be a "Change Station Name" verb input() which defaults to it.